packages/frontend/src/components/dialogs/SetupMultiDevice/ReceiveBackupProgressDialog.tsx

```diff
@@ -76,13 +84,24 @@ export function ReceiveBackupProgressDialog({
               {tx('error')}: {error}
             </p>
           )}
+          <p>
+            {importProgress === 0
+              ? tx('multidevice_connecting')
```

Comment:
i'd just use the existing string `connecting...` - this is already translated and usually, we avoid having double strings until we know that there will be conflicts:

```suggestion
              ? tx('connectivity_connecting')
```

(would habe been better if the key would be more neutral, but well, that's not the worse :)

Reply:
I also thought about this but decided not to comment because you could argue that that string could be translated as "Connecting to email server", especially that there is a translation comment for it.

But I won't insist.

Reply:
the "official" translation comment says "stay short", that's fine. we can separate the strings as soon as trouble appear, but in the vast majority of cases, over all the years, it is fine to reuse (not saying this approach is great for all apps, but in case of Delta Chat, it was :)